src/main/java/com/faforever/client/chat/ChannelTabController.java

```diff
@@ -59,10 +84,15 @@ public ChannelTabController() {
   public void setChannelName(String channelName) {
     super.setReceiver(channelName);
     this.channelName = channelName;
-
     channelTabRoot.setId(channelName);
     channelTabRoot.setText(channelName);
 
+
+    userSearchTextField.setPromptText(i18n.get("chat.userCount", chatService.getChatUsersForChannel(channelName).size()));
+    chatService.getChatUsersForChannel(channelName).addListener((MapChangeListener<? super String, ? super ChatUser>) change -> {
+      Platform.runLater(() -> userSearchTextField.setPromptText(i18n.get("chat.userCount", change.getMap().size())));
+    });
+
```

Comment:
In JavaFX, use `ChangeListener` when you're interested which element(s) was added or removed, but use an `InvalidationListener` if you just need to know when it has been updated.